Database/CreateTables.sql

```diff
@@ -38,10 +39,25 @@ CREATE TABLE IF NOT EXISTS Contacts (
 	email VARCHAR(100) NOT NULL DEFAULT '',
 	phone VARCHAR(20) NOT NULL DEFAULT '',
 	userID INT,
+	contactImageID INT,

```

Comment:
```suggestion
	contactImageID INT NULL,
```
contactImageID can and should be NULLable because there could be the case when user doesn't want to assign image to contact.